ingest_operations.py

```diff
@@ -363,8 +363,8 @@ def authorize_program_for_user(user_id):
 
     # we need to check to see if the program even exists in the system
     all_programs = auth.list_programs_in_opa(token)
-    if program_dict["program_id"] not in all_programs:
-        return {"error": f"Program {program_dict['program_id']} does not exist in {all_programs}"}
+    if program_dict["program_id"] not in all_programs[0]:
```

Comment:
I'd probably do `all_programs, status_code = authx.auth.list_programs_in_opa()` on the above line and then keep these the same, as that is going to be the change that is in develop now. That way there won't be a merge conflict later.

Reply:
ah good point thanks!